<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>331</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="dateEditEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>11</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Appointments</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableView" name="AppointmentView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AvailabilityButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Adjust Availability</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Availability</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="ThursdayLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Thursday: Not Available</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="SaturdayLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Saturday: Not Available</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="WednesdayLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wednesday: Not Available</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="SundayLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sunday: Not Available</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="FridayLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Friday: Not Available</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="MondayLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Monday: Not Available</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="TuesdayLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tuesday: Not Available</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="actionCreate_Account"/>
    <addaction name="actionChange_Password"/>
    <addaction name="actionDelete_Account"/>
    <addaction name="actionLog_Out"/>
   </widget>
   <widget class="QMenu" name="menuAppointment">
    <property name="title">
     <string>Appointment</string>
    </property>
    <addaction name="actionCreate_Appointment"/>
    <addaction name="actionChange_Appointment"/>
    <addaction name="actionCancel_Appointment"/>
    <addaction name="actionSend_Reminders"/>
    <addaction name="actionList_Appointment_in_Range"/>
   </widget>
   <widget class="QMenu" name="menuContact">
    <property name="title">
     <string>Contact</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionView_Contact_List"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuCalendar_Settings">
     <property name="title">
      <string>Calendar Settings</string>
     </property>
     <widget class="QMenu" name="menuCalendar_Color">
      <property name="title">
       <string>Calendar Color</string>
      </property>
      <addaction name="actionBlue"/>
      <addaction name="actionRed"/>
      <addaction name="actionPurple"/>
      <addaction name="actionDefault"/>
     </widget>
     <widget class="QMenu" name="menuFirst_Day_of_the_Week">
      <property name="title">
       <string>First Day of the Week</string>
      </property>
      <addaction name="actionSunday"/>
      <addaction name="actionMonday"/>
      <addaction name="actionTuesday"/>
      <addaction name="actionWednesday"/>
      <addaction name="actionThursday"/>
      <addaction name="actionFriday"/>
      <addaction name="actionSaturday"/>
     </widget>
     <addaction name="menuCalendar_Color"/>
     <addaction name="actionGridlines"/>
     <addaction name="menuFirst_Day_of_the_Week"/>
    </widget>
    <addaction name="menuCalendar_Settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContact_Admin"/>
    <addaction name="actionAbout_AppointmentCheck"/>
   </widget>
   <addaction name="menuAccount"/>
   <addaction name="menuAppointment"/>
   <addaction name="menuContact"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate_Account">
   <property name="text">
    <string>Create Account</string>
   </property>
  </action>
  <action name="actionDelete_Account">
   <property name="text">
    <string>Delete Account</string>
   </property>
  </action>
  <action name="actionLog_Out">
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
  <action name="actionCreate_Appointment">
   <property name="text">
    <string>Create Appointment</string>
   </property>
  </action>
  <action name="actionChange_Appointment">
   <property name="text">
    <string>Change Appointment</string>
   </property>
  </action>
  <action name="actionCancel_Appointment">
   <property name="text">
    <string>Cancel Appointment</string>
   </property>
  </action>
  <action name="actionAdd_New_Contact">
   <property name="text">
    <string>Add New Contact</string>
   </property>
  </action>
  <action name="actionChange_Contact_Info">
   <property name="text">
    <string>Change Contact Info</string>
   </property>
  </action>
  <action name="actionGridlines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gridlines</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionPurple">
   <property name="text">
    <string>Purple</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionSunday">
   <property name="text">
    <string>Sunday</string>
   </property>
  </action>
  <action name="actionMonday">
   <property name="text">
    <string>Monday</string>
   </property>
  </action>
  <action name="actionTuesday">
   <property name="text">
    <string>Tuesday</string>
   </property>
  </action>
  <action name="actionWednesday">
   <property name="text">
    <string>Wednesday</string>
   </property>
  </action>
  <action name="actionThursday">
   <property name="text">
    <string>Thursday</string>
   </property>
  </action>
  <action name="actionFriday">
   <property name="text">
    <string>Friday</string>
   </property>
  </action>
  <action name="actionSaturday">
   <property name="text">
    <string>Saturday</string>
   </property>
  </action>
  <action name="actionView_Contact_List">
   <property name="text">
    <string>Contact List</string>
   </property>
  </action>
  <action name="actionSend_Reminders">
   <property name="text">
    <string>Send Reminder</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
  <action name="actionContact_Admin">
   <property name="text">
    <string>Contact Admin</string>
   </property>
  </action>
  <action name="actionAbout_AppointmentCheck">
   <property name="text">
    <string>About AppointmentCheck...</string>
   </property>
  </action>
  <action name="actionList_Appointment_in_Range">
   <property name="text">
    <string>List Appointment in Range</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
